### PR TITLE
all: run gofmt -s -w from 1.19

### PR DESCRIPTION
**Pull request by keegancsmith**
gofmt in go1.19 does a lot of reformating of godoc strings, mostly to make them more consistent around lists.

Test Plan: CI

### CODE REVIEW

File: cmd/frontend/internal/highlight/highlight.go [hunk 1]
```diff
@@ -304,7 +304,7 @@ func (h *HighlightedCode) LinesForRanges(ranges []LineRange) ([][]string, error)
 	return lineRanges, nil
 }
 
-/// identifyError returns true + the problem code if err matches a known error.
+// / identifyError returns true + the problem code if err matches a known error.
```
Comment (burmudar) [inline cmd/frontend/internal/highlight/highlight.go:307]: Think we can lose the extra / ?

File: monitoring/definitions/shared/constructor.go [hunk 2]
```diff
@@ -95,7 +95,9 @@ func makeFilters(containerLabel, containerName string, filters ...string) string
 }
 
 // makeBy returns the suffix if the aggregator expression (e.g., max by (queue)),
-//                                                                   ^^^^^^^^^^
+//
+//	^^^^^^^^^^
+//
```
Comment (bobheadxi) [inline monitoring/definitions/shared/constructor.go:100]: I think the original here is better

File: internal/database/dbconn/open.go [hunk 3]
```diff
@@ -88,27 +88,29 @@ func openDBWithStartupWait(cfg *pgx.ConnConfig) (db *sql.DB, err error) {
...
 // (which the sqlHooks driver does not implement), extendedConn goes to the original default driver.
 //
-//                             Ping()
-//                             ResetSession()
-//                             CheckNamedValue()
-//                    ┌──────────────────────────────┐
-//                    │                              │
-//                    │                              │
-//                    │                              │
+//	         Ping()
```
Comment (burmudar) [inline internal/database/dbconn/open.go:91]: The formatter breaks the diagram here

**Review by burmudar — APPROVED** [review]
Small things really but overall 👍🏻

Is there a formatting directive to tell it to leave the docstring alone?

**Comment by keegancsmith** [comment]
This is the formatter that will automatically run when you run gofmt once on go1.19, so as people edit these files in the future it will do these edits. I will go through your comments as well as eyeball everything to fixup any bad formatting / fix it so it respects go's new rules around go docs.

File: enterprise/internal/insights/background/historical_enqueuer.go [hunk 4]
```diff
@@ -315,28 +315,27 @@ type RepoStore interface {
...
+//   - Check the commit index to see if any timeframes can be discarded (if they didn't change)
+//   - For each frame:
+//   - Find the oldest commit in the repository.
+//   - For every unique search insight series (i.e. search query):
+//   - Consider yielding/sleeping.
+//   - If the series has data for this timeframe+repo already, nothing to do.
+//   - If the timeframe we're generating data for is before the oldest commit in the repo, record a zero value.
+//   - Else, locate the commit nearest to the point in time we're trying to get data for and
+//     enqueue a queryrunner job to search that repository commit - recording historical data
+//     for it.
```
Comment (keegancsmith) [inline enterprise/internal/insights/background/historical_enqueuer.go:329]: According to https://tip.golang.org/doc/comment go does not support nested lists in comments. I'll see if I can redo this comment.

**Comment by keegancsmith** [comment]
Alright I eyeballed all the comments and pushed a bunch of changes. The only major change I had to do was for historical enqueuer since godoc does not support nested lists. Please take a look @valerybugakov https://github.com/sourcegraph/sourcegraph/pull/41629#discussion_r969240598

Otherwise if something is wrong we can followup.

**Pull request merged**